test/e2e/suite/suite.go

```diff
@@ -0,0 +1,47 @@
+// Copyright Jetstack Ltd. See LICENSE for details.
+package suite
+
+import (
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/jetstack/kube-oidc-proxy/test/e2e/framework"
+	"github.com/jetstack/kube-oidc-proxy/test/environment"
+)
+
+var (
+	env *environment.Environment
+	cfg = framework.DefaultConfig
+)
+
+var _ = SynchronizedBeforeSuite(func() []byte {
+	var err error
+	env, err = environment.Create(1, 3)
```

Comment:
Not too sure why we need 1 +  3 nodes for testing Kube-OIDC-Proxy

If this has a good reason comment it